src/main/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder.java

```diff
@@ -342,7 +342,9 @@ protected final String getSuccessfulDestination() {
         return triggers.toMap();
     }
 
+    @Restricted(NoExternalUse.class) // currently used only by jelly / stapler
     public List<TriggerDescriptor> getTriggerDescriptors() {
+        // TODO remove this once core has support for DescriptorVisibilityFilter on Trigger.for_(Item)
```

Comment:
As above, that is a misrepresentation of what I said.

* `for_` should not call `DescriptorVisibilityFilter`
* merely removing this would revert the fix; you would need to _add_ a `DescriptorVisibilityFilter` implementation